src/frontend/app/shared/components/list/list-types/services-wall/service-instance-card/service-instance-card.component.html

```diff
@@ -24,7 +24,18 @@
     <app-meta-card-key>Last Updated</app-meta-card-key>
     <app-meta-card-value>{{ serviceInstanceEntity?.entity.last_operation?.updated_at | date:'medium' }} </app-meta-card-value>
   </app-meta-card-item>
-
+  <app-meta-card-item>
+    <app-meta-card-key>Dashboard URL</app-meta-card-key>
+    <app-meta-card-value>
+      <div *ngIf="hasDashboardUrl(); then dashboardUrl; else none"></div>
```

Comment:
It's generally a bad idea to use functions in templates as these need to be re-evaluated every cycle - I know this seems hypocritical considering we're doing this for the other values but we shouldn't have used functions for most of those either (issue created #2531). In this case, I would directly use this.serviceInstanceEntity.entity.dashboard_url (or assign it to a value on the component) and use this for both the ngIf and the href value.

Reply:
Got it.